code/modules/economy/supply_misc.dm

```diff
@@ -116,6 +172,44 @@ ABSTRACT_TYPE(/area/supply)
 			if (prob(5))
 				qdel(src)
 
+/obj/strip_door/attackby(obj/item/I, mob/user)
+	if (ispryingtool(I))
+		playsound(src.loc, 'sound/items/Crowbar.ogg', 30, 1, -2)
+		src.change_direction()
+	if (isscrewingtool(I))
+		playsound(src.loc, 'sound/items/Screwdriver.ogg', 30, 1, -2)
+		SETUP_GENERIC_ACTIONBAR(user, src, 1 SECOND, /obj/strip_door/proc/screw_flaps, null, I.icon, I.icon_state, null, INTERRUPT_ACT | INTERRUPT_STUNNED | INTERRUPT_ACTION | INTERRUPT_MOVE)
```

Comment:
Not in favour of making these unanchorable.

Reply:
I'd really love a balance discussion on this, because when you construct and build these in practice, I could see it getting annoying really quickly to have to move them via deconstruction. I'd happily make the process a bit more involved to find a middle ground, even if it's just turning up the length of the animation, but for convenience I'd really like them unanchorable, like any other construction item like pipes or conveyor belts.

Reply:
Imo having these able to slide around would open up weirdness for little gain - like doors, I don't see much need to be able to drag these around at will

Reply:
Since you can place them now, being able to unachor them is just convenience. It's a simpler solution than having to grab a deconstruction device, grab a solderin iron, deconstruct them, scoot them over one tile, and then construct them. I'll remove it if it's really necessary, but you can already "slide them around", it just takes a lot more steps.